pkg/fsutil/file.go

```diff
@@ -33,6 +33,11 @@ func SafeMove(src, dst string) error {
 			return err
 		}
 
+		err = in.Close()
+		if err != nil {
+			return err
+		}
+
```

Comment:
This is redundant with the `defer in.Close()` call above. I think a better approach would be to convert the lines above `os.Remove` to be an inline function:
```
func() {
	in, err := os.Open(src)
	...
}()

if err := os.Remove(src) {
...
}
```

This way the deferred operations will be executed before the `Remove` is called. Can you make this change and ensure it fixes your original issue?
```

Reply:
I'll test that change but it may take a day or two, just FYI.